operatorconfig/driverconfig/powermax/v2.12.0/node.yaml

```diff
@@ -89,7 +89,7 @@ spec:
             capabilities:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
-          image: quay.io/dell/container-storage-modules/csi-powermax:v2.12.0
+          image: quay.io/dell/container-storage-modules/csi-powermax:nightly
           imagePullPolicy: IfNotPresent
```

Comment:
Please set the pull policy to 'Always' for all the nightly images

Reply:
updated